src/output.cpp

```diff
@@ -147,6 +155,30 @@ void Output::lhist_prepare(const std::vector<uint64_t> &values, int min, int max
   }
 }
 
+std::string Output::get_helper_error_msg(int func_id, int retcode) const
+{
+  std::string msg;
+  if (func_id == libbpf::BPF_FUNC_map_update_elem && retcode == -E2BIG)
+  {
+    msg = "Map full; can't update element. Try increasing MAP_KEYS_MAX.";
```

Comment:
This is the "old" name for w/ the config changes. Sounds like there's consensus on changing the env var names - should we preemptively use BPFTRACE_MAX_MAP_KEYS here?

Reply:
Depends on the merging order :) This is a smaller change so I imagine this will get merged first. Happy to change it with the config PR.

Reply:
I think this is good to merge. Then the other PR can rebase on top